corehq/apps/locations/static/locations/js/location_drilldown.async.js

```diff
@@ -79,7 +79,7 @@ function LocationSelectViewModel(hierarchy, default_caption, auto_drill, loc_fil
 
     // load location hierarchy and set initial path
     this.load = function(locs, selected) {
-        this.root(new model.func({name: '_root', children: locs}, this));
+        this.root(new model.func({name: '_root', children: locs}, this, undefined, undefined, undefined, loc_url));
```

Comment:
fwiw, for JS, i try to usually have separate args for every required arg, and only pass in options for things that are options